server/app/models/wish.rb

```diff
@@ -1,5 +1,9 @@
 class Wish < ApplicationRecord
   belongs_to :api_user
+  belongs_to :programminglanguage
```

Comment:
you can do that with https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-belongs_to

it's a bit confusing, but can be done with just the `belongs_to` with `class_name` and `foreign_key` if I am not mistaken.

Reply:
Unsure what you mean, 'belongs_to' is used above, what changes should be made here?